src/pyodide/python-entrypoint-helper.ts

```diff
@@ -270,6 +274,16 @@ function makeEntrypointClass(className: string, classKind: AnyClass): any {
       return new Proxy(this, makeEntrypointProxyHandler(pyInstancePromise));
     }
   };
+
+  // Add dummy functions to the class so that the validator can detect them.
```

Comment:
Is the point that these will never get accessed on an instance because of the `Proxy`? If so could you explain in the comment?

Reply:
Yep.